changes to build with the updated libgweather which only has one
argument to gweather_info_new.

This should be deleted when evolution-data-server is updated to 3.34.0

--- a/src/calendar/backends/weather/e-weather-source.c	2019-12-05 12:43:18.996481246 -0800
+++ b/src/calendar/backends/weather/e-weather-source.c	2019-12-05 12:17:04.622428784 -0800
@@ -185,8 +185,8 @@
 
 	if (source->priv->info == NULL) {
 		source->priv->info = gweather_info_new (
-			source->priv->location,
-			GWEATHER_FORECAST_LIST);
+			source->priv->location
+	        );
 		gweather_info_set_enabled_providers (source->priv->info, GWEATHER_PROVIDER_ALL);
 		g_signal_connect (
 			source->priv->info, "updated",
